code/datums/components/art.dm

```diff
@@ -40,8 +41,4 @@
 	to_chat(M, "You start examining [parent].")
 	if(!do_after(M, 20, target = parent))
 		return
-	on_obj_examine(source, M)
-
-#undef BAD_ART
-#undef GOOD_ART
-#undef GREAT_ART
+	on_obj_examine(source, M)
```

Comment:
you need those undefs probably

Reply:
I got rid of the undefs so that the defines can actually be used for reasons above

Reply:
yeah well you're breaking the linter by reusing macros